app/src/app.config.json

```diff
@@ -1028,14 +1028,16 @@
       {
         "id": "createdDateRange",
         "name": "SEARCH.SEARCH_HEADER.FILTERS.DATE.TITLE",
-        "columnKey": "createdAt",
```

Comment:
Is it intentional?

Reply:
No this was not intentional. I've restored it